libr/include/r_util/r_annotated_code.h

```diff
@@ -74,110 +75,99 @@ typedef struct r_code_annotation_t {
 		} variable;
 	};
 } RCodeAnnotation;
-
+/**
+ * \struct r_annotated_code_t r_annotated_code.h libr/include/r_util/r_annotated_code.h
+ * 
+ * \brief This structure contains the decompiled code and all the annotations for the decompiled code.
+ */
 typedef struct r_annotated_code_t {
-	char *code; // owned
-	RVector /*<RCodeAnnotation>*/ annotations;
+	char *code; /**< Decompiled code. RAnnotatedCode owns this string and it must free it. */ 
+	RVector /*<RCodeAnnotation>*/ annotations; /**< RVector<RCodeAnnotation> contains the list of annotations for the decompiled code. */ 
 } RAnnotatedCode;
 
 /**
- * r_annotated_code_new() - Creates a new RAnnotatedCode structure and returns its pointer.
- * @code: Literal code for which the RAnnotatedCode structure will be created .
- *
- * This functions creates a new RAnnotatedCode structure.
- * RAnnotatedCode.code will be initialized as the character array passed.
- * Here, code must be a string that can deallocated.
+ * This function creates and initializes a new RAnnotatedCode
+ * structure with the specified decompiled code that's passed
+ * as an argument. Here, the argument code must be a string that can be deallocated.
  * This will initialize RVector<RCodeAnnotation> annotations as well.
  * 
- * Return: Pointer to the new RAnnotatedCode structure created.
+ * @brief Create and initialize a RAnnotatedCode structure and returns its pointer.
+ * 
+ * @param code A deallocatable character array.
+ * @return Pointer to the new RAnnotatedCode structure created.
  */
 R_API RAnnotatedCode *r_annotated_code_new(char *code);
 /**
- * r_annotated_code_free() - Deallocates *code.
- * @code: Pointer to a RAnnotatedCode.
- *
- * This functions deallocates memory allocated for *code.
+ * @brief Deallocates the dynamically allocated memory for the specified RAnnotatedCode.
  * 
- * Return: Nothing.
+ * @param code Pointer to a RAnnotatedCode.
  */
 R_API void r_annotated_code_free(RAnnotatedCode *code);
 /**
- * r_annotation_free() - Deallocates dynamically allocated memory for the specified annotation.
- * @e: Pointer to the annotation
- * @user: Always NULL for this function. Present here for this function to be of the type RVectorFree.
- * 
  * This function recognizes the type of the specified annotation and
  * frees memory that is dynamically allocated for it.
+ * @brief Deallocates dynamically allocated memory for the specified annotation.
  * 
- * Return: Nothing.
+ * @param e Pointer to the annotation.
+ * @param user Always NULL for this function. Present here for this function to be of the type RVectorFree.
```

Comment:
When mentioning other types like `RVectorFree` you can mark them using `@ref` like this `@ref RVectorFree`. It will cause doxygen to insert a link to the documentation of corresponding element.

![Screenshot from 2020-08-04 12-06-13](https://user-images.githubusercontent.com/7101031/89276109-e751a400-d64b-11ea-97dd-029c12974f97.png)
![Screenshot from 2020-08-04 12-12-39](https://user-images.githubusercontent.com/7101031/89276113-e7ea3a80-d64b-11ea-8bdd-cda882c78182.png)

Reply:
Done.